invenio_vocabularies/config.py

```diff
@@ -164,5 +164,14 @@
 """ORCID summaries bucket name."""
 VOCABULARIES_ORCID_SYNC_MAX_WORKERS = 32
 """ORCID max number of simultaneous workers/connections."""
-VOCABULARIES_ORCID_SYNC_DAYS = 1
-"""ORCID number of days to sync."""
+VOCABULARIES_ORCID_SYNC_SINCE = {
+    "years": 0,
+    "months": 0,
+    "weeks": 0,
+    "days": 1,
+    "hours": 0,
+    "minutes": 0,
+    "seconds": 0,
+    "microseconds": 0,
```

Comment:
```suggestion
    "days": 1,
```

nit: I think we can do with just the one we need, and just write in the docstring that it's parameters passed to `timedelta` constructor.

Reply:
Ok, makes sense 👍